### ISSUE TITLE
Add examples of Language on a Linguistic Object

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#144]
Add examples with Language on a LinguisticObject

**Comment by azaroth42** [comment]
(Unduplicated #144 and #154 by making this one only language)

**Comment by azaroth42** [comment]
Blocked by #110.

**Comment by azaroth42** [comment]
Unblocked by closing #110 as `wontfix`.
